src/components/NcButton/index.ts

```diff
@@ -1,7 +1,7 @@
-/**
+/*!
```

Comment:
Typo?

Reply:
No `/**` is interpreted as documentation but this is legal information which has somewhat "standard" `/*!`.
The benefit of doing so: Some toolings will recognize the comment block as license. AND it will not be shown in docs (e.g. when using Typedoc / JSDoc).